.env.example

```diff
@@ -29,6 +29,7 @@ OPEN_AI_WHISPER_API_KEY=YOUR_API_KEY
 # Only "ELEVEN_LABS" for now
 TEXT_TO_SPEECH_USE=ELEVEN_LABS
 ELEVEN_LABS_API_KEY=YOUR_API_KEY
+GOOGLE_TTS_SERVICE_ACCOUNT_FILE=YOUR_JSON_FILE
```

Comment:
question, can TTS and STT use the same account file? see line 25

Reply:
Yes, the same service account key file can be used for both TTS and STT in Google Cloud, but the TTS api needs to be enabled. I will remove the redundant line. 
Regarding the implementation of Google's TTS, should we follow the current httpx approach as it is in elevenlabs or use the texttospeech.TextToSpeechClient library from Google Cloud as it is in Google STT?

Reply:
I think as long as the TTS interface is the same, it's okay to have different implementation inside